packages.config

```diff
@@ -1,19 +1,19 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<packages>
-  <package id="DotNetNuke.Core" version="9.5.0" targetFramework="net472" />
-  <package id="DotNetNuke.DependencyInjection" version="9.5.0" targetFramework="net472" />
-  <package id="DotNetNuke.Web" version="9.5.0" targetFramework="net472" />
-  <package id="DotNetNuke.Web.Client" version="9.5.0" targetFramework="net472" />
-  <package id="DotNetNuke.Web.Mvc" version="9.5.0" targetFramework="net472" />
-  <package id="DotNetNuke.WebApi" version="9.5.0" targetFramework="net472" />
-  <package id="Microsoft.AspNet.Mvc" version="5.1.3" targetFramework="net462" />
-  <package id="Microsoft.AspNet.Razor" version="3.1.2" targetFramework="net462" />
-  <package id="Microsoft.AspNet.WebApi.Client" version="5.2.3" targetFramework="net462" />
-  <package id="Microsoft.AspNet.WebApi.Core" version="5.2.3" targetFramework="net462" />
-  <package id="Microsoft.AspNet.WebPages" version="3.1.2" targetFramework="net462" />
-  <package id="Microsoft.Extensions.DependencyInjection" version="2.1.1" targetFramework="net462" />
-  <package id="Microsoft.Extensions.DependencyInjection.Abstractions" version="2.1.1" targetFramework="net462" />
-  <package id="Microsoft.Web.Infrastructure" version="1.0.0.0" targetFramework="net462" />
-  <package id="MSBuildTasks" version="1.5.0.235" targetFramework="net472" developmentDependency="true" />
-  <package id="Newtonsoft.Json" version="10.0.3" targetFramework="net462" />
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="DotNetNuke.Core" version="9.5.0" targetFramework="net472" />
+  <package id="DotNetNuke.DependencyInjection" version="9.5.0" targetFramework="net472" />
+  <package id="DotNetNuke.Web" version="9.5.0" targetFramework="net472" />
+  <package id="DotNetNuke.Web.Client" version="9.5.0" targetFramework="net472" />
+  <package id="DotNetNuke.Web.Mvc" version="9.5.0" targetFramework="net472" />
+  <package id="DotNetNuke.WebApi" version="9.5.0" targetFramework="net472" />
+  <package id="Microsoft.AspNet.Mvc" version="5.1.3" targetFramework="net472" />
+  <package id="Microsoft.AspNet.Razor" version="3.1.2" targetFramework="net472" />
+  <package id="Microsoft.AspNet.WebApi.Client" version="5.2.3" targetFramework="net472" />
+  <package id="Microsoft.AspNet.WebApi.Core" version="5.2.3" targetFramework="net472" />
+  <package id="Microsoft.AspNet.WebPages" version="3.1.2" targetFramework="net472" />
+  <package id="Microsoft.Extensions.DependencyInjection" version="2.1.1" targetFramework="net472" />

```

Comment:
The `targetFramework` is set by the NuGet package and should not be changed manually, correct?